selection_sort/Java/SelectionSort.java

```diff
@@ -0,0 +1,54 @@
+import java.util.Random;
+
+/*
+ ____       _           _   _             ____             _   
+/ ___|  ___| | ___  ___| |_(_) ___  _ __ / ___|  ___  _ __| |_ 
+\___ \ / _ | |/ _ \/ __| __| |/ _ \| '_ \\___ \ / _ \| '__| __|
+ ___) |  __| |  __| (__| |_| | (_) | | |  ___) | (_) | |  | |_ 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpz6y7xvz3/selection_sort/Java/SelectionSort.java
+++ b/tmp/tmpz6y7xvz3/selection_sort/Java/SelectionSort.java
@@ -4,7 +4,7 @@
  ____       _           _   _             ____             _   
 / ___|  ___| | ___  ___| |_(_) ___  _ __ / ___|  ___  _ __| |_ 
 \___ \ / _ | |/ _ \/ __| __| |/ _ \| '_ \\___ \ / _ \| '__| __|
- ___) |  __| |  __| (__| |_| | (_) | | |  ___) | (_) | |  | |_ 
+ ___) |  __| |  __| (__| |_| | (_) | | |  ___) | (_) | |  | |_
 |____/ \___|_|\___|\___|\__|_|\___/|_| |_|____/ \___/|_|   \__|
 
 */
```